clients/python-wrapper/lakefs/branch.py

```diff
@@ -32,15 +32,22 @@ def create(self, source_reference_id: str | Reference, exist_ok: bool = False) -
         """
         Create a new branch in lakeFS from this object
 
+        Example of creating a new branch:
```

Comment:
Don't we want to place the example with the operation?

Reply:
The example is of create branch in the docsting of Branch.create - am I missing something?